dash/dash.py

```diff
@@ -725,7 +739,9 @@ def index(self, *args, **kwargs):  # pylint: disable=unused-argument
         config = self._generate_config_html()
         metas = self._generate_meta_html()
         renderer = self._generate_renderer()
-        title = getattr(self, "title", "Dash")
+
+        # use self.title instead of app.config.title for backwards compatibility
```

Comment:
In principle we could make title into a `@property` that syncs `app.title` with `app.config.title` - but not a big deal. Most people should just use the kwarg regardless.